abis/mlibc/utsname.h

```diff
@@ -0,0 +1,13 @@
+#ifndef _ABIBITS_UTSNAME_T_H
+#define _ABIBITS_UTSNAME_T_H
+
+struct utsname {
+	char sysname[65];
+	char nodename[65];
+	char release[65];
+	char version[65];
+	char machine[65];
+	// char domainname[65]; Uncommenting this makes the ABI gods unhappy.
```

Comment:
where does this field come from?

also if it's not appropriate to add it, just drop it and we can add it to abi-break later

Reply:
`uname` for some platforms like linux includes that field (https://man7.org/linux/man-pages/man2/uname.2.html), hidden behind a define, the plan was uncommenting it in a following pr to abi-break.

Reply:
I'm not aware of any use in practice. We could merge without it and follow up on abi-break to bunch it in whenever the next break happens.